autoload/post-form.php

```diff
@@ -24,6 +24,7 @@ public function postForm($request) {
       )
     ) {
       $_POST = $params;
+      do_action("post_submission_filter", $params);
```

Comment:
Prefix the action with the name of the plugin, e.g. `municipio_gatsby_post_form`